extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/JaegerPropagator.java

```diff
@@ -110,6 +110,9 @@
     Objects.requireNonNull(getter, "getter");
 
     SpanContext spanContext = getSpanContextFromHeader(carrier, getter);
+    if (!spanContext.isValid()) {
```

Comment:
can you add unit test cases for this new behavior, for each of the propagators that's changed?

Reply:
Oh my, didn't realize I haven't written those ones. Will do super early tomorrow.